go/vt/vttablet/tabletserver/planbuilder/plan.go

```diff
@@ -43,60 +40,47 @@ var (
 // PlanType indicates a query plan type.
 type PlanType int
 
+// The following are PlanType values.
```

Comment:
Most of these are self-evident, but for a few of them, I would not mind comments with explanation